pyproject.toml

```diff
@@ -39,7 +39,7 @@ dependencies = [
   'PyYAML~=6.0',
   'requests~=2.32.0',
   'segment-analytics-python~=2.2.2',
-  'sentence-transformers==2.*',
+  'sentence-transformers~=2.6.1',
```

Comment:
We don't really need to update the `pyproject.toml` as soon as one of our dep has a sec-vul. If we have too strict rules this can lead for dependency hell for the users, and it's the user responsibility to ensure the environment is up to date from a security perspective.

Reply:
thanks Gonéri; in this case I think it's what we want (newer than 2.6.1 but in the same family) and it's consistent with (most) of the other entries but your broader point about pyproject v requirements* is a good one.

Can we update this and the rest of our dependencies in a new PR when we've defined a standard for it?